Resources/doc/reference/routing.rst

```diff
@@ -107,6 +107,37 @@ explicitly this defaults to the action name.
         }
     }
 
+Make use of all route parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the ``add`` method create a Symfony ``Route``, you can use all constructor parameters of the ``Route`` as parameters
+in the ``add`` method to set additional settings like for example methods.
+
+.. code-block:: php
+
+    <?php
+    use Sonata\AdminBundle\Route\RouteCollection;
+
+    class MediaAdmin extends Admin
+    {
+        protected function configureRoutes(RouteCollection $collection)
+        {
+            $collection->add(
+                'custom_action',
+                $this->getRouterIdParameter().'/custom-action'
+                array(),
```

Comment:
I don't know if it's better in one line, but ok.
I have no examples where the other parameters are make sense. But someone else might. I think it's ok when it's empty, because that illustrates only more, that it's possible to leave them empty.